pkg/scalers/rabbitmq_scaler.go

```diff
@@ -37,6 +37,8 @@ const (
 	rabbitMetricType                       = "External"
 	rabbitRootVhostPath                    = "/%2F"
 	rmqTLSEnable                           = "enable"
+	rabbitHttpQueryMessageRateAge          = "messageRatesAge"
+	rabbitHttpQueryMessageRatesIncrement   = "messageRatesIncrement"
```

Comment:
Let's improve the name here to make more sense, apart from Rabbit's shitty name (IMO)
```suggestion
	rabbitHttpQueryMessageRateAge          = "messageRateAge"
	rabbitHttpQueryMessageRatesIncrement   = "messageRateIncrement"
```